src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Main program for Duke.
+ */
+public class Duke {
+    /**
+     * Main program function.
+     * @param args input arguments, not used
+     */
+    public static void main(String[] args) {
+        Ui.printGreeting();
+        String userInput;
+        TaskList userTaskList = new TaskList();
+        Scanner scanner = new Scanner(System.in);
+        loop: while (true) {
```

Comment:
For the handling of commands, can consider creating separate command classes with an 'execute' method that executes the logic for that command. In that way, you can abstract out all the command logic from the main method and the main method can just focus on getting parsing user input and the big switch statement can be removed.